### ISSUE TITLE
Hovers don't seem to work

### ISSUE THREAD

**Issue by BoykoAlex** [gtache/intellij-lsp#15]
The documentation hovers (not diagnostic marker hovers) see to only work for TEXT files?
https://github.com/gtache/intellij-lsp/blob/master/intellij-lsp/resources/META-INF/plugin.xml#L69

I've tried making these hovers for `java` files. Contributed the `LSPDocumentationProvider` for `JAVA` language. This seem to trigger breakpoints in `LSPDocumentationProvider`, but I still never see a single hover request sent to the LS.

Can you please provide more details on how to get documentation hovers working and what are the limitations with hovers?

**Comment by BoykoAlex** [comment]
I think the hovers only if I open the file with appropriate extension in the `TEXT` editor. Only then i see the hovers... Any idea why hovers don't work in the Java editor? Does Java support doesn't let the `LSPDocumentationProvider` to kick in at all?

**Comment by gtache** [comment]
Yes, as Java is natively supported by IntelliJ, I let the native support manage everything. It's the same with the other languages, if there is already a plugin supporting a language, I'll let it answer any action it can without interfering with it.    
I'll perhaps introduce an option to still let the requests go through, but you'll end up with two windows for the hover for example (one from IntelliJ, one from the server).
